src/components/version-context-switcher/types.ts

```diff
@@ -14,14 +12,14 @@ export interface VersionContextSwitcherProps {
 	/**
 	 * (optional) The initial value of the switcher when first rendered.
 	 */
-	initialValue?: ContextSwitcherOption['value']
+	initialValue?: ContextSwitcherOption
 
 	/**
 	 * (optional) A function invoked when the value of the switcher has changed.
 	 * It is invoked after the component has run its internal code for handling
 	 * changes to the switcher.
 	 */
-	onChange?: ChangeEventHandler<HTMLSelectElement>
+	onChange?: (x: string) => void
```

Comment:
```suggestion
	onChange?: (version: string) => void
```
💭 Not sure if this is the right suggestion, but a more descriptive parameter would be good!